flight/Modules/VtolPathFollower/vtol_follower_control.c

```diff
@@ -627,6 +529,142 @@ int32_t vtol_follower_control_attitude(float dT)
 	return 0;
 }
 
+
+static float loiter_deadband(float input) {
+	const float CMD_THRESHOLD = 0.2f;
+
+	if (input > CMD_THRESHOLD) {
+		input -= CMD_THRESHOLD;
+	} else if (input < -CMD_THRESHOLD) {
+		input += CMD_THRESHOLD;
+	} else {
+		input = 0;
+	}
+
+	input /= (1 - CMD_THRESHOLD);	// Normalize to -1 to 1 range.
+
+	/* Confine to desired range */
+	if (input > 1.0f) {
+		input = 1.0f;
+	} else if (input < -1.0f) {
+		input = -1.0f;
+	}
+
+	return expo3(input, 40);	// And apply 40% expo
```

Comment:
still for this one we should so people can play with that feel and just make the default 40 if you feel that is a good value